### PR TITLE
Some readme work, added ssh ingress, etc.

### PR DESCRIPTION
**Pull request by Valien**
@dgkirkwood - this PR adds an `ingress` rule for SSH for troubleshooting as well as some README clarifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,23 +38,21 @@ The great thing about Machine ID is that it can helpe with use cases outside of
...
 2. Create a role for your Machine ID bot. This bot should have permissions to only manage tokens and nothing else. An example of a role with these privileges is at `teleport/gha_bot_role.yaml`. Apply the role to your cluster using `tctl create -f gha_bot_role.yaml`
 3. Create the Machine ID bot. This will link a bot user to the role and token we just created. `tctl bots add gha-token-bot --roles=<match your role name> --token=<match your token name>`
 
 ### Github Actions Configuration
 1. The example workflow at `.github/workflows/main.yaml` requires four secrets to run. 
-   * PAT:  A github Personal Access Token which will give the workflow permissions to check out your repository if it is private. 
+   * PAT:  A github Personal Access Token which will give the workflow permissions to check out your repository if it is private. Even if your repo is public you will still need this token for the workflow to run properly.
```
Comment (dgkirkwood) [inline README.md]: Lets just make this:
A github Personal Access Token which will give the workflow permissions to check out your repository.

**Review by dgkirkwood — COMMENTED** [review]
Hey Allen - all great suggestions. One comment to change wording slightly and we are good to go on the README. 
You did mention the working dir did not work for you, but cd did. Do you want to include that in this PR as well?

File: README.md (hunk 1 above)
Comment (Valien) [reply README.md]: done!

**Comment by Valien** [comment]
> Hey Allen - all great suggestions. One comment to change wording slightly and we are good to go on the README. You did mention the working dir did not work for you, but cd did. Do you want to include that in this PR as well?

Went ahead and added it to this PR. I was still doing some local testing on it. Seems to work well in my env now.

**Comment by Valien** [comment]
@dgkirkwood - fixed the README and added the `cd terraform` parts into the GHA workflow. Should still be in this same PR.

**Review by dgkirkwood — APPROVED** [review]
All looks good thank you Allen!

**Pull request merged**